packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -7,12 +7,16 @@
 import React, {ReactElement} from 'react'
 // @ts-ignore
 import {CommerceApiProvider} from 'commerce-sdk-react'
+// @ts-ignore
+import {useCorrelationId} from 'pwa-kit-react-sdk/ssr/universal/contexts'
```

Comment:
Why do we have 2 definitions of this hook? one in the contexts and one in the hooks index? 

Side note: is it weird that we are importing a hook from the context folder when there is explicitly a hooks folder where other hooks are imported from?

Reply:
I tend to prefer having the context hook be in the same place with its Provider rather than having it in a different place. I think it is better to find the hook whenever I need to change anything related to Provider. In case we have a large number of hooks like the Retail React App, I find it really hard to look for the context hooks among other hooks. However, this is just a preference, not super important in this package since we don't have a lot of hooks in general. I've moved the hook to the hook folder.